instant.ts

```diff
@@ -149,21 +191,36 @@ const setEnvVars = (packageInfo: PackageInfo) => {
 
     envVars.push({
       'Environment Variable': envVar,
-      'Default Value': defaultEnv,
-      'Updated Value': env[envVar]
+      'Current Value': env[envVar]
     })
   }
 
   if (envVars?.length > 0) {
-    console.table(envVars)
+    console.log(
+      `🛠️ Config set for ${packageInfo.metadata.name} (${packageInfo.metadata.id}):`
+    )
+    console.table(
+      envVars.map(
+        ({ 'Environment Variable': envVar, 'Current Value': currVal }) => ({
+          'Environment Variable':
+            envVar.length > 50
+              ? `${envVar.substring(0, 50)}...[trunc]`
```

Comment:
Minor: Suggestions:
Could it be valuable to instead of only showing the first 50 chars, and cutting off the rest, to rather should first 25chars, then ellipses, and then the last 25chars?

Im thinking of a usecase where the value was changes but only the last couple of chars were updated, but this would be lost if only using the beginning of the string

Reply:
We could but then we might miss changes made in the middle. It's hard as we don't really know where the changes were made. Also, we need to make it obvious that it was truncated and someone may miss something in a middle of a string but it's quite obvious when it's at the end.